.github/workflows/linux.yml

```diff
@@ -1,6 +1,25 @@
 name: Linux builds
 
-on: [push, pull_request]
+on:
+  push:
+    paths-ignore:
+      - '.clang-format'
+      - '.mdl-styles'
+      - '*.md'
+      - 'docs/**'
+      - 'licenses/**'
+      - 'website/**'
```

Comment:
contrib/resources/translations/** is IMHO, another good candidate.

Reply:
Cheers, adding.

Reply:
Ok, I think we must rebuild whenever any file in the bundled resources folder changes.